ballerina-tests/tests/03_bidirectional_streaming_service.bal

```diff
@@ -50,29 +50,23 @@ boolean initialized = false;
 }
 service "Chat" on ep3 {
 
-    remote function chat(ChatStringCaller caller, stream<ChatMessage, error?> clientStream) {
+    remote function chat(ChatStringCaller caller, stream<ChatMessage, error?> clientStream) returns error? {
         log:printInfo(string `${caller.getId()} connected to chat`);
         connectionsMap[caller.getId().toString()] = caller;
-        log:printInfo("Client registration completed. Connection map status");
-        log:printInfo("Map length: " + connectionsMap.length().toString());
-        log:printInfo(connectionsMap.toString());
         initialized = true;
-        error? e = clientStream.forEach(function(ChatMessage chatMsg) {
+        checkpanic clientStream.forEach(function(ChatMessage chatMsg) {
```

Comment:
```suggestion
        check clientStream.forEach(function(ChatMessage chatMsg) {
```

Reply:
Fixed in https://github.com/ballerina-platform/module-ballerina-grpc/pull/606/commits/c32ef7e98e2bfaa4df966a96025d16d755dfb1cd